src/components/AutoSuggest/AutoSuggest.tsx

```diff
@@ -289,7 +278,7 @@ const AutoSuggest: FC<AutoSuggestProps> = ({
         role="combobox"
       >
         <StyledInput
-          aria-activedescendant={activeId}
+          aria-activedescendant={activeId.toString() ?? null}
           aria-autocomplete="list"
```

Comment:
this can never be null, right? since on line 82 activeId is always defined

Reply:
Good catch. I forgot to revert that change.